src/main/java/com/example/b2b_opportunities/Entity/Skill.java

```diff
@@ -0,0 +1,34 @@
+package com.example.b2b_opportunities.Entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@RequiredArgsConstructor
+@Table(name = "skills")
+public class Skill {
+    @Id
+    @Column(name = "id", nullable = false)
+    private String id;
+
+    @ManyToOne
+    @JoinColumn(name = "parent_id")
+    private Skill parent;
+
+    @NotNull
+    private String name;
+
+    @NotNull
+    private Boolean assignable;
+    private String image;
```

Comment:
Are we going to store the images as strings or blob/byte array?

Reply:
All images are around 500-600Kb in total (99 icons), which is nothing really. I've already implemented storing those images in the DB. Checked prices for Amazon storage and that will be around $0.00005 per month. I will upload it as it's already done. We can modify it later if needed.